modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing and upgrading the {pipelines-title} Operator.
+*** To install a specific version of the {pipelines-title} Operator, cluster adminsitrators can also the `pipelines-<version>` channel. For example, to install the {pipelines-title} Operator version `1.8`, you can use the `pipelines-1.8` channel.
 +
 [NOTE]
 ====
-The `preview` and `stable` channels will be deprecated and removed in a future release.
+The `preview` and `stable` are not available for installation and upgrade of the Operator.
```

Comment:
In 4.9 and 4.10, the `stable` and `preview` channels are still present. It is not present in 4.11 and above.
cc: @concaf @ppitonak @Vincent056 
![Screenshot from 2022-11-07 15-40-08](https://user-images.githubusercontent.com/13780748/200464966-347da68a-d186-4821-85d1-934bb0830de2.png)